thali/thalireplicationmanager.js

```diff
@@ -112,6 +148,18 @@ ThaliReplicationManager.prototype.stop = function () {
   });
 };
 
+function informDeviceIdentityListeners(err, publicKeyHash) {
+  this._deviceIdentityListeners.forEach(function (listener) {
+    listener(err, publicKeyHash);
+  });
+}
+
+function clearDeviceIdentityListeners() {
+  while(this._deviceIdentityListeners.length > 0) {
```

Comment:
Learned a new thing today :). Also removed the function as it only has one line now.